Documentation/compatibility/GetECDsaPublicKey-correctly-handles-brainpool-curves.md

```diff
@@ -0,0 +1,25 @@
+## ECDsaCertificateExtensions.GetECDsaPublicKey correctly handles Brainpool curves
+
+### Scope
+Minor
+
+### Version Introduced
+4.8
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+Starting with the .NET Framework 4.8, the `System.Security.Cryptography.X509Certificates.ECDsaCertificateExtensions.GetECDsaPublicKey(this System.Security.Cryptography.X509Certificates.X509Certificate2 certificate)` will now correctly handle Brainpool elliptic curves.
```

Comment:
While "Brainpool" is fine for the title, really the change was to handle all named curves that Windows 10 supports (that Windows 8.1 did not).

So perhaps "will now correctly handle all named curves supported by the OS, including the Brainpool elliptic curves."